packages/server/lib/util/ts-node.js

```diff
@@ -27,6 +27,7 @@ const registerTsNode = (projectRoot, pluginsFile) => {
     debug('typescript path: %s', tsPath)
     debug('registering project TS with options %o', tsOptions)
 
+    require('tsconfig-paths/register')
```

Comment:
does `ts-node` not expose similar functionality? seems a bit weird that it wouldn't